test/basic.html

```diff
@@ -84,12 +84,31 @@
     });
 
     suite('pasting', function(done) {
+      function ensureDocumentHasFocus() {
+        window.top && window.top.focus();
+      }
+
+      // On IE10 and IE11 pasting pops up an "Allow access" dialog, which
+      // we can't bypass (and means we can't run the tests).
+      var pasteRequiresUserInteraction;
+      setup(function() {
+        pasteRequiresUserInteraction = !document.queryCommandSupported('insertText');;
```

Comment:
We don't use `paste`, we use `insertText`, which drops the formatting: https://github.com/PolymerElements/paper-text-field/blob/master/paper-text-field.html#L118

I thought the test should test for the thing we're actually going to be using :)